assets/javascripts/kitten/components/form/checkbox.test.js

```diff
@@ -34,4 +34,18 @@ describe('Checkbox with default props', () => {
       expect(input).to.have.attr('id', 'input-1')
     })
   })
+
+  describe('renders children', () => {
+    const componentChildren = shallow(
+      <Checkbox>
+        <svg />
+      </Checkbox>
+    )
+
+    it('it children', () => {
```

Comment:
On peut éviter de répéter le `it` ici, en disant par exemple `it('accepts children', () => {`.